### PR TITLE
Add Task, interactions and search parameters to AU eReq Server CapabilityStatement (FHIR-48915)

### PR DESCRIPTION
**Pull request by JayMurdoch**
Fix for [FHIR-48915](https://jira.hl7.org/browse/FHIR-48915)

* Add Task to Server CapStat as SHALL support
* Add interactions as described
* Add search parameters as described
* Change log entry

### CODE REVIEW

**Review by michael-wilson-au — APPROVED** [review]

File: input/resources/au-erequesting-server.xml [hunk 1]
```diff
@@ -349,5 +349,176 @@
...
+      <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+        <valueCode value="SHALL"/>
+      </extension>
+      <type value="Task"/>
+      <supportedProfile value="http://hl7.org.au/fhir/ereq/StructureDefinition/au-erequesting-task">
+        <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+          <valueCode value="SHALL"/>
+        </extension>
+      </supportedProfile>
+      <documentation value="Servers **SHALL** support the Task resource, the AU eRequesting profile, and the conformance expectations for the Task resource.&#xA;&#xA;The Server **SHALL** support the `_include` parameters for `Task:patient`, `Task:requester`, `Task:owner`, `Task:focus`."/>
```
Comment (dbojicic) [inline input/resources/au-erequesting-server.xml:363]: 'parameters' should be singular 'parameter'.

File: input/pagecontent/changes.md [hunk 2]
```diff
@@ -20,3 +20,6 @@ This change log documents the significant updates and resolutions implemented fr
 - Added _include search parameters ServiceRequest:patient, ServiceRequest:requester and ServiceRequest:encounter to AU eRequesting Placer, Filler, Patient and Server CapabilityStatements [FHIR-46942](https://jira.hl7.org/browse/FHIR-46942)
 - Renamed AU eRequesting Patient Access actor and CapabilityStatement to be AU eRequesting Patient [FHIR-46802](https://jira.hl7.org/browse/FHIR-46802)
 - Changed the conformance requirement for ServiceRequest create and update interactions from SHALL to SHOULD in AU eRequesting Server CapabilityStatement [FHIR-47141](https://jira.hl7.org/browse/FHIR-47141)
+- Made the following changes to AU eRequesting Server CapabilityStatement:
+  - Added the following search parameters as SHALL: _id, group-identifier, owner, patient, requester, focus, status, _lastUpdated, patient+status, owner+status, requester+status, _lastUpdated+status+owner, patient.identifier, owner.identifier, requester.identifier, Task:patient, Task:requester, Task:owner and Task:focus [FHIR-48915](https://jira.hl7.org/browse/FHIR-48915)
```
Comment (dbojicic) [inline input/pagecontent/changes.md:24]: please change to lower case 'addded'

File: input/pagecontent/changes.md [hunk 3]
```diff
@@ -20,3 +20,6 @@ This change log documents the significant updates and resolutions implemented fr
...
+  - Added Task interactions read and search-type as SHALL, and create and update as SHOULD [FHIR-48915](https://jira.hl7.org/browse/FHIR-48915)
```
Comment (dbojicic) [inline input/pagecontent/changes.md:25]: please change to lower case 'addded'

**Review by dbojicic — CHANGES_REQUESTED** [review]
Please see comments

File: input/resources/au-erequesting-server.xml (hunk 1 above)
Comment (JayMurdoch) [reply input/resources/au-erequesting-server.xml:363]: Amended, thanks.

File: input/pagecontent/changes.md (hunk 2 above)
Comment (JayMurdoch) [reply input/pagecontent/changes.md:24]: Amended, thanks.

File: input/pagecontent/changes.md (hunk 3 above)
Comment (JayMurdoch) [reply input/pagecontent/changes.md:25]: Amended, thanks.

**Review by dbojicic — APPROVED** [review]

**Pull request merged**